spec/jenkins_pipeline/app_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+
+describe JenkinsPipeline::App do
+
+  it "Returns urls from all pipeline config files" do
+    get '/api/pipelines/ids'
+    # expect(subject).to have_received(:configuration)
```

Comment:
I don't know if it was on purpose but it looks like this line would be the main point of this test once the rest of the test is asserting on the configuration of the request.